pages/news-and-views/_slug/index.vue

```diff
@@ -12,37 +14,55 @@ export default {
 		const linkList = store.state.posts.map(({ slug }) => slug)
 		return linkList.includes(params.slug)
 	},
-	head () {
-		return {
-			title: this.Title
-		}
-	},
+	// head () {
+	// 	return {
+	// 		title: this.Title
+	// 	}
+	// },
 	components: {
 		PostArticle
 	},
 	data () {
 		return {
 		}
 	},
-	computed: {
-		links () {
-			return this.$store.state.posts.map(({ slug }) => slug)
-		},
-		linkNames () {
-			return this.links.map(x => x.replace(/-/g, " "))
-		},
-		Title () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
-		},
-		Picture () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
-		},
-		Content () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
-		},
-		Tags () {
-			return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
-		}
+	// computed: {
+	// 	links () {
+	// 		return this.$store.state.posts.map(({ slug }) => slug)
+	// 	},
+	// 	linkNames () {
+	// 		return this.links.map(x => x.replace(/-/g, " "))
+	// 	},
+	// 	Title () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).title
+	// 	},
+	// 	Picture () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).picture
+	// 	},
+	// 	Content () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).content
+	// 	},
+	// 	Tags () {
+	// 		return this.$store.state.posts.find(post => post.slug === this.$route.params.slug).tags
+	// 	}
+	// },
+	asyncData (context) {
+		return axios.get(`${Config.wpDomain}${Config.apiBase}posts`).then((response) => {
+			const defaultPicture = "https://wecount.inclusivedesign.ca/wp-content/uploads/2019/10/We-Count-logos_colour-and-bw-01.png"
+			const res = response.data.filter(post => post.slug === context.params.slug)[0]
+			// let altTag = ""
+			const api = res._links["wp:featuredmedia"][0].href
+			const pic = axios.get(api)
```

Comment:
If you use the `_embed` parameter (docs: https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_embed) on the initial axios request, you won't need to perform a second request to retrieve the featured image. This could provide some performance improvements, as all information would be retrieved in a single request.

Reply:
For an example of what this does, take a look at the JSON response here: https://wecount.inclusivedesign.ca/wp-json/wp/v2/posts/79?_embed